### PR TITLE
Adding DHT, LCD, Servo & Distance sensor to easygopigo

### PR DESCRIPTION
**Pull request by Shoban94**

### CODE REVIEW

File: Software/Python/easygopigo.py [hunk 1]
```diff
@@ -674,6 +679,217 @@ def take_photo(self,filename):
...
+            print("Temperature = %.02fC Humidity = %.02f%%"%(temp, humidity))
+            return [temp, humidity] 
+             
+    # Derived from Robert's Code
+    # function which eliminates the noise
+    # by using a statistical model
+    # we determine the standard normal deviation and we exclude anything that goes beyond a threshold
+    # think of a probability distribution plot - we remove the extremes
+    # the greater the std_factor, the more "forgiving" is the algorithm with the extreme values
+    def eliminateNoise(self,values, std_factor = 2):
```
Comment (CleoQc) [inline Software/Python/easygopigo.py]: rename this one

**Comment by DexterInd** [comment]
@Shoban94 and @CleoQc this branch is now conflicting.  What do you want to do with it?

**Comment by Shoban94** [comment]
@DexterInd , This might be conflicting now because of the new changes made to easygopigo by other PRs. So when merging this PR, you might have to take my work separately out and add it the latest script and send a new PR.

**Comment by CleoQc** [comment]
I think we can close this PR. It's been here a while and the code took a different turn

**Comment by DexterInd** [comment]
Closed.  Thanks for the reminder!

**Pull request closed without merging**